### PR TITLE
Development of Awesome books using classes

### PR DESCRIPTION
**Pull request by Sami-Ullah-Tufail**
## The change of method to Classes 

We changed the style of the Books layout
We added the Classes in the js code

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+addButton.addEventListener("click", () => {
+  bookList.addBook(bookTitleInput.value, bookAuthorInput.value);
+  bookList.renderBooks(document.getElementById("bookList"));
+  bookTitleInput.value = "";
+  bookAuthorInput.value = "";
+});
+
+// Render the existing books on page load
+bookList.renderBooks(document.getElementById("bookList"));
```
Comment (mckent05) [inline index.js]: **OPTIONAL** - Good job done so far, but to help make your project more readable, I suggest rendering the book when the `DOM` loads, i.e use the event listener `DOMContentLoaded`. Please fix this 🙏

File: index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable */
```
Comment (mckent05) [inline index.js]: - You have done well so far, as much as you are allowed to disable eslint for some rules, it is not advisable to disable the eslint for the whole file. Please remember that eslint helps us write clean code that meets global standards, please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Sami-Ullah-Tufail, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job refactoring your code to use classes 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      li.appendChild(removeButton);
+      this.bookListElement.appendChild(li);
+    });
+  }
+}
+// eslint-disable-next-line no-new
+document.addEventListener('DOMContentLoaded', () => {
+  // eslint-disable-next-line no-new
+  new BookList();
+});
```
Comment (Munsa1) [inline index.js]: - [ ] Good job removing the `eslint-disable` at the top of the file, however, you have introduced two new `eslint-disable-next-line no-new` comments. Kindly note that the suggestion by the previous [reviewer here](https://github.com/Sami-Ullah-Tufail/Awesome-books/pull/3#discussion_r1288230483) to render the book when the DOM loads, left by the previous reviewer was optional and if it is causing an error kindly fix it.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Sami-Ullah-Tufail @Yonasle, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Implemented Optional Suggestions :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Sami-Ullah-Tufail @Yonasle 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Great use of JavaScript class ✔️ 
- Application works properly ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**